pages/concepts/bank-funding-source.mdx

```diff
@@ -0,0 +1,149 @@
+| [Dwolla + Plaid integration](https://developers.dwolla.com/guides/plaid#plaid-integration)                                    | Yes                                                     | Online banking credentials                 | ~85%              | Checking, <br /> Savings                                     | [Tokenized integration](https://www.dwolla.com/integrations/plaid/)                                                                                                               |
+| Other Approved Third-party Provider                                                                                           | Yes, if part of third party offering                    | Variable                                   | Variable          | Variable                                                     | Variable                                                                                                                                                                          |
+
+As you decide on the various methods for adding a funding-source, a good thing to keep in mind is that a transfer between two parties requires the sending party to have a verified bank account. Bank account verification prior to sending funds is required by the ACH network.
+
+## Bank Addition + Verification methods
+
+### Add a Bank via the API and verify using Microdeposits
+
+**The use case:**
+I want to add a U.S. bank account by supplying an account and routing number. The bank will be unverified until microdeposits are initiated and verified.
+
+This is commonly implemented for receive-only “payout” use cases where users are receiving funds, but not sending.
+
+**Tell me more**
+Using the Dwolla API you can add an unverified funding source to a [Customer](https://docs.dwolla.com/#create-a-funding-source-for-a-customer) as well as your [Master Account](https://docs.dwolla.com/#create-a-funding-source-for-an-account). Additionally, you can manually add an unverified bank funding source to your Master Dwolla Account or your end users directly from the [Dwolla Dashboard](https://www.dwolla.com/platform/dashboard/).
+
+<img
+  src={addFundingSourceForm}
+  alt="An example form UI component to present to your users."
+  width="400"
+/>
+
+**Ready to build?**
+Adding a bank with the Dwolla API is easy. Take a look at our documentation to learn more about this process. After adding a bank via the API, you can leave it unverified if you will only be sending funds to it. If you or your user will be sending funds from the bank account, you can verify it using microdeposits.
+
+<InlineCTA
+  icon={guideIcon}
+  text="Check our step-by-step guide on verifying a bank using microdeposits."
```

Comment:
Could we remove the anchor here on the link? #verify-a-bank-account-with-micro-deposits